profiles/permissions.py

```diff
@@ -26,10 +26,18 @@ def has_permission(self, request, view):
         return request.user == view._profile.person
 
 
+class OnlyRequestUserOrAdmin(BasePermission):
```

Comment:
I don't see the Admin checking here, maybe it would be better to change name of this permission class

I see the logic which checks if method in safe methods and if not then checks if user try to get own user info. So it would be better to name this like OwnerOrReadOnly

Reply:
if request.method in SAFE_METHODS and user.is_staff

user.is_staff is admin user

Reply:
or OwnerOrAdmin

Reply:
Thank you for feedback. Done.